slack/commands/rate.go

```diff
@@ -0,0 +1,73 @@
+package commands
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"log"
+
+	"github.com/sjuls/soup-ranking/dbctx"
+
+	"github.com/sjuls/soup-ranking/score"
+	"github.com/sjuls/soup-ranking/utils"
+)
+
+type (
+	rateFlags struct {
+		Score   int
+		Comment string
+	}
+
+	rateCommand struct {
+		repo score.Repository
+	}
+)
+
+// NewRateCommand create a new rate command
+func NewRateCommand(repo score.Repository) Command {
+	var rateCmd Command = &rateCommand{repo}
+	return rateCmd
+}
+
+func (c *rateCommand) Execute(args []string, output io.Writer) {
+	flags, err := extractRateFlags(args, output)
+	if err != nil {
+		fmt.Fprintln(output, err.Error())
+		return
+	}
+
+	if 1 > flags.Score || flags.Score > 10 {
```

Comment:
Since we're checking that `Score` is _not_ between 1 and 10, I would reorder this: `flags.Score < 1 || flags.Score > 10`